League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs

```diff
@@ -26,7 +28,20 @@ public leagueSandboxAutoSetupForm()
 
             this.FormClosing += (_, _2) =>
             {
-                Environment.Exit(0);
+                try
+                {
+                    _abortInitiated = true;
+                    abortText.Visible = true;
+                    startButton.Text = "Start";
+                    startButton.Enabled = false;
```

Comment:
Why do you need to change variables before the form close?

Reply:
If you clicked the close button and it threw an exception. Once completed.